src/structures/CategoryChannel.js

```diff
@@ -7,6 +7,14 @@ const GuildChannel = require('./GuildChannel');
  * @extends {GuildChannel}
  */
 class CategoryChannel extends GuildChannel {
+  constructor(guild, data) {
+    super(guild, data);
+
+    // Category Channels cannot have invites
+    this.createInvite = undefined;
```

Comment:
Is there even any need for this? You moved the base function from GuildChannel to TextChannel, CategoryChannel inherits from GuildChannel, not TextChannel

Reply:
If you’re referring to my setting of those methods to undefined, this was a solution to another problem brought up about category channels improperly inheriting methods https://github.com/discordjs/discord.js/pull/4832#issuecomment-695605097

Reply:
Oops, missed that, you're right